lib/spack/spack/cmd/list.py

```diff
@@ -40,6 +40,16 @@ def setup_parser(subparser):
         nargs=argparse.REMAINDER,
         help="optional case-insensitive glob patterns to filter results",
     )
+    subparser.add_argument(
+        "-r",
+        "--repo",
+        "-N",
+        "--namespace",
+        dest="repos",
```

Comment:
This displays as:
```
  -r REPOS, --repo REPOS, -N REPOS, --namespace REPOS
                        only list packages from the specified repo/namespace
```
which might be mistaken as allowing plural in a single call, somehow, so maybe:
```suggestion
        dest="repo",
```
and add ` (multiple use allowed)` as in:
```
  -t TAG, --tag TAG     filter a package query by tag (multiple use allowed)
```